packages/serverless/ci/testing/pipeline-debug.yaml

```diff
@@ -0,0 +1,73 @@
+---
+
+resources:
+
+  - name: nodejs-repository
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/instana/nodejs.git
+      branch: node20-dev
+      username: ((instanacd-github-api-token))
+      password: x-oauth-basic
+
+
+  - name: instana-aws-fargate-npm-package
+    type: npm-resource
+    icon: cube-outline
+    source:
+      scope: instana
+      package: aws-fargate
```

Comment:
Is this specific to aws-fargate? As its for local testing we can name it generic?

Reply:
This resource downloads the fargate npm package. The testing pipeline is just a minified version of the real serverless pipeline. We can't rename this.